modules/client/src/tokenVoting/internal/utils.ts

```diff
@@ -232,6 +237,21 @@ function parseToken(
       name: subgraphToken.name,
       type: TokenType.ERC721,
     };
+  } else if (subgraphToken.__typename === SubgraphContractType.ERC20_WRAPPER) {
+    token = {
+      address: subgraphToken.id,
+      symbol: subgraphToken.symbol,
+      name: subgraphToken.name,
+      decimals: subgraphToken.decimals,
+      type: TokenType.ERC20,
+      underlyingToken: {
```

Comment:
Don't we have a way for consumers to check the enum to determine that this is a wrapper? If checking whether `underlyingToken` exists is fine, then resolve the comment.

Reply:
Yep, thats basically the way to do it, if underliyingToken exists is wrapped, else it isnt